src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -8,11 +8,14 @@ struct EnDoor;
 typedef void (*EnDoorActionFunc)(struct EnDoor*, GlobalContext*);
 
 typedef struct EnDoor {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x63];
-    /* 0x01A7 */ s8 unk_1A7;
-    /* 0x01A8 */ char unk_1A8[0x20];
-    /* 0x01C8 */ EnDoorActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ char unk_144[0x5C];
```

Comment:
```suggestion
    /* 0x144 */ UNK_TYPE1 unk_144[0x5C];
```

Reply:
I don't mind changing them, but wouldn't this be left up to whoever is decomping `EnDoor` when they fill out the struct completely?

Reply:
Yeah this one is my mistake. I thought it was already decomped and you were changing something that part of the struct.